enclave-runtime/Cargo.lock

```diff
@@ -4136,7 +4160,7 @@ dependencies = [
 [[package]]
 name = "sp-std"
 version = "4.0.0"
-source = "git+https://github.com/paritytech/substrate.git?branch=master#40703f41167c718e13b491228efeafb4a6194448"
+source = "git+https://github.com/paritytech/substrate.git?branch=master#4262c63fecf5de9a459775d772329828bcb549c8"
```

Comment:
I think it is weird that sp-std has changed on both lock-files, no?

Reply:
Ah, I ran a cargo update, because it didn't compile at first. Shall I revert?

Reply:
Ah sorry I was confused. This is the single substrate-branch=master dependency, so this is fine. 👍